test/utils/charts.go

```diff
@@ -0,0 +1,77 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/onsi/gomega"
+	chartloader "helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/chartutil"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	"github.com/gardener/landscaper/pkg/utils/simplelogger"
+)
+
+// WaitForHelmDeployItemToSucceed waits for a DeployItem to succeed
+func WaitForHelmDeployItemToSucceed(ctx context.Context, logger simplelogger.Logger, kubeClient client.Client, objKey types.NamespacedName, timeout time.Duration) error {
```

Comment:
This function could be used for any kind of deploy items not only helm charts?

Reply:
Yep, I will rename it and move it to `test/utils/landscaper.go`.